test/number.js

```diff
@@ -331,6 +333,7 @@ describe('number', () => {
 		[
 			{ locale: 'ar', expect: ['42 %', '-42 %'] },
 			{ locale: 'ar-SA', expect: ['42 %', '-42 %'] },
+			{ locale: 'cy-GB', expect: ['42 %', '-42 %'] },
```

Comment:
Welsh looks to not have the space between number and % (https://www.unicode.org/cldr/cldr-aux/charts/25/summary/cy.html#3303)

Reply:
Hmmm... they're gonna hate my `d2l-input-percent`.  😬

Reply:
Or maybe they will ❤️ it, and everyone else will dislike.  🤷‍♂️

Reply:
😱

Reply:
So again, I went with [what the monolith has](https://search.d2l.dev/xref/lms/lp/_dbschema/D2L.LP.Globalization.Locale/Data/LOCALE_CULTURE.xml?r=4bed0d06#494) which has the space. I'm not sure what's "right" and I've raised issues against deviations in the monolith before.

Reply:
Haha the only people who will hate your percent input Dave are those who put the percent first -- `% 9`... which is just Turkish out of the locales we support.